lib/glific/notion.ex

```diff
@@ -114,7 +114,7 @@ defmodule Glific.Notion do
         "rich_text" => [
           %{
             "type" => "text",
-            "text" => %{"content" => "#{registration.org_details["current_address"]}"}
+            "text" => %{"content" => inspect(registration.org_details["current_address"])}
```

Comment:
Because the address we are getting here is a map and interpolating a map here was giving error
`protocol String.Chars not implemented`
instead of interpolation we can also use:
`Enum.map(address, fn {key, value} -> "#{key}: #{value}" end)
  |> Enum.join(", ")`

Reply:
@AmishaBisht Can we use the approach you mentioned here. Since using `inspect` kind of put escape characters to the stringified map. Won't it be better if the data in DB is readable?